packages/waku/src/router/define-router.ts

```diff
@@ -629,3 +630,12 @@ export function unstable_defineRouter(fns: {
 
   return defineHandlers({ handleRequest, handleBuild });
 }
+
+let publicRouterConfig: any | undefined;
```

Comment:
I think `any` is just temporary, but `MyConfig` is very internal, which is not intended to export. We'll likely refactor it.

I think it's better to expose the result of `getConfig`. 🤔